documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -68,29 +69,29 @@ We request OpenAPI(Swagger) spec authoring be assigned to engineers who have an
 | Id | Rule Name | Output Message |Severity | 
 | --- | --- | --- | --- |
 | M2026 | AvoidAnonymousTypes, AnonymousBodyParameter |	Inline/anonymous models must not be used, instead define a schema with a model name in the "definitions" section and refer to it. This allows operations to share the models. | Error |
-| M2014	| OperationParametersValidation	| Parameter "subscriptionId" is not allowed in the operations section, define it in the global parameters section instead | Error |
-| M2027	| DefaultMustBeInEnum | The default value is not one of the values enumerated as valid for this element. | Error |
+| S2014	| OperationParametersValidation	| Parameter "subscriptionId" is not allowed in the operations section, define it in the global parameters section instead | Error |
```

Comment:
Is `OperationParametersValidation` the right name in the source code? and do we still have rules with id prefix `S` int he code after sarangan's changes?